From c783b29cbed5275f22b15a02e4fd6faac675eeb1 Mon Sep 17 00:00:00 2001
From: Pradeep Sriram <pradeep.sriram@amlogic.com>
Date: Thu, 4 Jun 2020 17:00:24 -0700
Subject: [PATCH] RDK: only compile amavutils. Others not needed for RDK

PD#SWPL-24384

Problem:
 libplayer is too heavy with many deps

Solution:
 only compile utils

Verify:
 Local

Change-Id: If7987b94de1bc3a780007a62886038947fa91770
Signed-off-by: Pradeep Sriram <pradeep.sriram@amlogic.com>
---
 Makefile           | 46 +++++++++++++++++++++++-----------------------
 amavutils/Makefile |  2 +-
 2 files changed, 24 insertions(+), 24 deletions(-)

diff --git a/Makefile b/Makefile
index 506ba48..7fae264 100755
--- a/Makefile
+++ b/Makefile
@@ -2,31 +2,31 @@
 
 #export CC BUILD_DIR STAGING_DIR TARGET_DIR
 all:
-	mkdir -p $(shell pwd)/../../staging/usr/lib/libplayer
-	-$(MAKE) -C amadec all
-	-$(MAKE) -C amadec install
+	#mkdir -p $(shell pwd)/../../staging/usr/lib/libplayer
+	#-$(MAKE) -C amadec all
+	#-$(MAKE) -C amadec install
 	-$(MAKE) -C amavutils all		
 	-$(MAKE) -C amavutils install
-	-$(MAKE) -C amffmpeg all		
-	-$(MAKE) -C amffmpeg install
-	-$(MAKE) -C amcodec all
-	-$(MAKE) -C amcodec install
-	-$(MAKE) -C amvdec all
-	-$(MAKE) -C amvdec install
-	-$(MAKE) -C audio_codec all
-	-$(MAKE) -C audio_codec install
-	-$(MAKE) -C amplayer all
-	-$(MAKE) -C amplayer install
-	-$(MAKE) -C streamsource all
-	-$(MAKE) -C streamsource install
-	-$(MAKE) -C examples all
-	-$(MAKE) -C third_parts/libcurl-ffmpeg all
-	-$(MAKE) -C third_parts/libdash all
-	-$(MAKE) -C third_parts/libvhls all
-	-$(MAKE) -C bootplayer all
-	-$(MAKE) -C bootplayer install
-	-$(MAKE) -C audioplayer all
-	-$(MAKE) -C audioplayer install
+	#-$(MAKE) -C amffmpeg all		
+	#-$(MAKE) -C amffmpeg install
+	#-$(MAKE) -C amcodec all
+	#-$(MAKE) -C amcodec install
+	#-$(MAKE) -C amvdec all
+	#-$(MAKE) -C amvdec install
+	#-$(MAKE) -C audio_codec all
+	#-$(MAKE) -C audio_codec install
+	#-$(MAKE) -C amplayer all
+	#-$(MAKE) -C amplayer install
+	#-$(MAKE) -C streamsource all
+	#-$(MAKE) -C streamsource install
+	#-$(MAKE) -C examples all
+	#-$(MAKE) -C third_parts/libcurl-ffmpeg all
+	#-$(MAKE) -C third_parts/libdash all
+	#-$(MAKE) -C third_parts/libvhls all
+	#-$(MAKE) -C bootplayer all
+	#-$(MAKE) -C bootplayer install
+	#-$(MAKE) -C audioplayer all
+	#-$(MAKE) -C audioplayer install
 
 install:
 	-$(MAKE) -C examples install
diff --git a/amavutils/Makefile b/amavutils/Makefile
index 5254dcf..e1451a2 100755
--- a/amavutils/Makefile
+++ b/amavutils/Makefile
@@ -8,7 +8,7 @@ TARGET=libamavutils.so
 
 INSTALL_DIR=${LIB_DIR}
 
-LDFLAGS += -shared -lpthread -lm -lrt
+CFLAGS += -shared -lpthread -lm -lrt
 
 TOPDIR=$(shell pwd)
 SRC?=$(TOPDIR)
-- 
2.7.4

